Sources/RegularPlayer.swift

```diff
@@ -169,7 +169,7 @@ extension AVMediaSelectionOption: TextTrackMetadata {
     }
     
     private func setupAirplay() {
-        #if os(iOS) || os(tvOS)
+        #if canImport(UIKit)
```

Comment:
Ah, specifically didn't call this one out since it's accessing this property on `AVPlayer`, from `AVFoundation`. I think that the straight `os(...)` checks were appropriate here (or, alternatively, an `if #available` check.